### PR TITLE
Simplify README and move code to example.

### PR DESCRIPTION
**Pull request by maltoe**
## TODOs

- [x] finish the loading of the styles into a module variable (or do this with `render` as well?)
- [x] add a b64-encoded image for demonstration

### CODE REVIEW

**Review by andreasknoepfle — APPROVED** [review]
Nice changes so far. I also love that the example is tested ❤️

**Comment by maltoe** [comment]
@andreasknoepfle ty! i added some boilerplate for asset loading.. not sure how much of this we actually should put into the example, as it grows more and more complex. But I wanted to show somehow how to include static assets (images as data urls as you mentioned, but also stylesheets) in a PDF template, so maybe it has to be this complex? Plus it's of course only one possible way to do this out of a million, I'm worried this "example" would have too much influence on users? wdyt? maybe we can have a call on this at some point?

**Comment by andreasknoepfle** [comment]
> But I wanted to show somehow how to include static assets (images as data urls as you mentioned, but also stylesheets) in a PDF template, so maybe it has to be this complex? 

Would it help breaking it into multiple examples? 

> Plus it's of course only one possible way to do this out of a million, I'm worried this "example" would have too much influence on users? 

I wouldn't worry about that too much. If someone gets the gist of it they will derive from this path soon enough :D. But often it is best to have those examples around as long as they all work and are not just abandoned code in some file (which I think is covered already by the testing part) 

> maybe we can have a call on this at some point?

Sure :)

File: examples/phoenix/phoenix_example.ex [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  @moduledoc false
+
+  def pdf do
+    quote do
+      use Phoenix.View, root: "examples/phoenix/templates"
+      import Phoenix.HTML
+      import PhoenixExample
+
+      def print_to_pdf(assigns, callback) do
+        [content: content(assigns)]
```
Comment (maltoe) [inline examples/phoenix/phoenix_example.ex:11]: page size?

File: examples/phoenix/assets/style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+img {
+  width: 400px;
+  height: 200px;
```
Comment (maltoe) [inline examples/phoenix/assets/style.css:3]: closing }

**Pull request merged**